src/main/java/calculator/PositiveNumber.java

```diff
@@ -0,0 +1,41 @@
+package calculator;
+
+public class PositiveNumber {
+    private final int value;
+
+    private PositiveNumber(int value) {
+        validate(value);
+        this.value = value;
+    }
```

Comment:
```suggestion
    private PositiveNumber(String value) {
        this(Integer.parseInt(value);
    }
    
    private PositiveNumber(int value) {
        validate(value);
        this.value = value;
    }
```
from 메서드 대신 위와 같이 String을 인자로 받는 부 생성자를 추가하는 것은 어떨까?